### PR TITLE
Fix docs of load_inference_model&load_params&load_persistables&chain.

### PR DESCRIPTION
**Pull request by wzzju**
## load_inference_model
image
image

## load_params
image

## load_persistables
image

## chain
image

### CODE REVIEW

File: python/paddle/fluid/io.py [hunk 1]
```diff
@@ -706,33 +706,38 @@ def name_has_fc(var):
...
+    the file ``filename``.
+
+    Use the ``dirname`` to specify the directory where parameters were saved. If
+    parameters were saved in separate files under the directory `dirname`, set
+    ``filename`` as None; if all parameters were saved in a single file, use
+    ``filename`` to specify the file name.
+
+    **Note**:
+        Some variables are not Parameter while they are necessary for
+        training, such learning rate, global step, etc. So you cannot save and
```
Comment (Superjomn) [inline python/paddle/fluid/io.py]: such as

File: python/paddle/reader/decorator.py [hunk 2]
```diff
@@ -117,19 +117,51 @@ def data_reader():
...
+        ``paddle.fluid.io.chain`` is recommended to use.
+
+    For example, if three input readers' outputs are as follows:
+    [0, 0, 0],
+    [10, 10, 10],
+    [20, 20, 20].
     The chained reader will output:
-    [0, 0, 0, 1, 1, 1, 2, 2, 2]
+    [0, 0, 0][10, 10, 10] [20, 20, 20].
+
```
Comment (Superjomn) [inline python/paddle/reader/decorator.py:133]: [[0, 0, 0][10, 10, 10] [20, 20, 20]]?

File: python/paddle/fluid/io.py (hunk 1 above)
Comment (wzzju) [reply python/paddle/fluid/io.py]: Done.

File: python/paddle/reader/decorator.py (hunk 2 above)
Comment (wzzju) [reply python/paddle/reader/decorator.py:133]: Done.

File: python/paddle/fluid/io.py (hunk 1 above)
Comment (wzzju) [reply python/paddle/fluid/io.py]: image

File: python/paddle/reader/decorator.py (hunk 2 above)
Comment (wzzju) [reply python/paddle/reader/decorator.py:133]: image

**Review by Superjomn — APPROVED** [review]
LGTM

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**